html/captive-portal/templates/violations/redirect.html

```diff
@@ -1 +1 @@
-<meta http-equiv="refresh" content="0; url=[% redirect_url %]" />
+<meta http-equiv="refresh" content="0; url=[% IF redirect_url %][% redirect_url %][% END %]" />
```

Comment:
If there is no redirect url where should it go packetfence.org or will there always be a redirect_url?

Reply:
I can set packetfence.org by default